orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/CreateServerGroupStage.groovy

```diff
@@ -35,12 +36,23 @@ class CreateServerGroupStage extends AbstractDeployStrategyStage {
 
   @Override
   protected List<TaskNode.TaskDefinition> basicTasks(Stage stage) {
-    return [
+    def tasks = [
       new TaskNode.TaskDefinition("createServerGroup", CreateServerGroupTask),
       new TaskNode.TaskDefinition("monitorDeploy", MonitorKatoTask),
       new TaskNode.TaskDefinition("forceCacheRefresh", ServerGroupCacheForceRefreshTask),
+      ]
+
+    if (true) {
```

Comment:
@anotherchrisberry This should check the `/features/stages` endpoint in clouddriver and only add the `tagServerGroup` task if the `upsertEntityTags` stage is enabled.

The `if (true)` was just a quick hack used to test.